[2021-07-20 20:20:18,848] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Fuse.fuse6 2021-07-19T00:05:00+00:00 [queued]>
[2021-07-20 20:20:21,213] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Fuse.fuse6 2021-07-19T00:05:00+00:00 [queued]>
[2021-07-20 20:20:21,214] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-20 20:20:21,214] {taskinstance.py:1068} INFO - Starting attempt 1 of 8
[2021-07-20 20:20:21,214] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-20 20:20:22,020] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): fuse6> on 2021-07-19T00:05:00+00:00
[2021-07-20 20:20:22,026] {standard_task_runner.py:52} INFO - Started process 1202442 to run task
[2021-07-20 20:20:22,039] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'Fuse', 'fuse6', '2021-07-19T00:05:00+00:00', '--job-id', '336', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpyxx2osdz', '--error-file', '/tmp/tmpox0qsiw7']
[2021-07-20 20:20:22,041] {standard_task_runner.py:77} INFO - Job 336: Subtask fuse6
[2021-07-20 20:20:22,577] {logging_mixin.py:104} INFO - Running <TaskInstance: Fuse.fuse6 2021-07-19T00:05:00+00:00 [running]> on host deepika-Inspiron-5570
[2021-07-20 20:20:22,842] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sthadpka93@gmail.com
AIRFLOW_CTX_DAG_OWNER=postgres
AIRFLOW_CTX_DAG_ID=Fuse
AIRFLOW_CTX_TASK_ID=fuse6
AIRFLOW_CTX_EXECUTION_DATE=2021-07-19T00:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-19T00:05:00+00:00
[2021-07-20 20:20:29,555] {python.py:151} INFO - Done. Returned value was: None
[2021-07-20 20:20:29,559] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=Fuse, task_id=fuse6, execution_date=20210719T000500, start_date=20210720T143518, end_date=20210720T143529
[2021-07-20 20:20:29,840] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-07-20 20:20:29,867] {local_task_job.py:151} INFO - Task exited with return code 0
